frontend/src/Choose.js

```diff
@@ -16,9 +16,9 @@ const recipelst = [
     {title: "Pizza", 
     description: 'Tastes like newspaper pizza, try it for yourself!', 
     body: 'lorem',
-    image: pizza_img, id:3} 
-]; 
-
+    image: pizza_img, id:3}   
+];            
+//hi
```

Comment:
Hi

Reply:
thanks for noticing that! I just took it out :)